### PR TITLE
Use new Scene api for textures

### PR DESCRIPTION
**Pull request by makiam**

### CODE REVIEW

File: ArtOfIllusion/src/artofillusion/image/ImagesDialog.java [hunk 1]
```diff
@@ -757,6 +745,7 @@ private void selectNone()
       for (BCheckBox b: removeBox) b.setState(false);
     }
 
+    @SuppressWarnings("ResultOfObjectAllocationIgnored")
```
Comment (Sailsman63) [inline ArtOfIllusion/src/artofillusion/image/ImagesDialog.java:748]: What line specifically triggers this warning? As a general rule, it's best to apply 'suppress warnings' for as narrow a chunk of code as possible. (As little as a single statement, when we can.)

**Review by Sailsman63 — CHANGES_REQUESTED** [review]

File: ArtOfIllusion/src/artofillusion/image/ImagesDialog.java (hunk 1 above)
Comment (makiam) [reply ArtOfIllusion/src/artofillusion/image/ImagesDialog.java:748]: this method creates instance of PurgeDialog which is not assigned to any variable. Just opens dialog

Comment (Sailsman63) [reply ArtOfIllusion/src/artofillusion/image/ImagesDialog.java:748]: Then the 'suppress warnings' goes down above line 761, to apply only to that statement...

Comment (makiam) [reply ArtOfIllusion/src/artofillusion/image/ImagesDialog.java:748]: Annotation cannot be applied to single line of code.

Comment (Sailsman63) [reply ArtOfIllusion/src/artofillusion/image/ImagesDialog.java:748]: You may have to add a local variable to assign the new PurgeDialog to... Which will probably stop the warning outright...

**Comment by Sailsman63** [comment]
Thanks!

**Pull request merged**